### ISSUE TITLE
NZB results limited to 5

### ISSUE THREAD

**Issue by poo706** [8cylinder/tv-overlord#60]
It appears as though tvol is only showing the 5 most recent results of a usenet indexer, even though there are 6 or more results if I search directly on the indexer's website.  There isn't a built in way to change this, right?  Is that controlled by line 42 of nzb.py?

**Comment by 8cylinder** [comment]
Yeah, that should be set to the `--count` setting not hard coded.

**Comment by poo706** [comment]
Are you saying that the option is already there?
